docs/xostor/xostor.md

```diff
@@ -311,6 +311,10 @@ RemainAfterExit=true
 - At least 1 disk on any machine of the pool (case without replication). Otherwise, any number of disks can be used on a machine. However, to be consistent, we recommend using the same model and number for each machine that has disks.
 - The replication/place count must be equal to 1, 2 or 3.
 
+:::warning
+- LINSTOR services like satellites and controller can use a lot of memory resources. It is therefore more than necessary to have sufficient RAM allocated to the Dom-0, 16 GiB can be enough for average pools with around a hundred volumes. But it may be essential to increase the dedicated memory for more intensive use. So make sure to monitor the memory usage of your pool to prevent the OOM Killer from being triggered.
```

Comment:
Maybe explain *why* it uses so much RAM, what you in exchange for it, so that it looks more like the right price to pay for the feature, rather than a drawback?

Reply:
Also, can we estimate the amount of RAM needed depending on the setup?

Reply:
I'm not sure I have a really good wording to indicate why it consumes so much. Also it is complicated to evaluate the right amount of RAM, it actually varies from one pool to another. We have users who have no problem with 8GiB (small pool with 3 machines), others with 16 GiB seems to be the acceptable minimum.